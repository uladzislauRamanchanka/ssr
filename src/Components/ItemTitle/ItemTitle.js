import React from "react";
import classes from "./ItemTitle.module.css";

const ItemTitle = (props) => (
  <div className={classes.ItemTitle}>{props.name}</div>
);

ItemTitle.defaultProps = {
  name: 'Default Title'
}

export default ItemTitle;
