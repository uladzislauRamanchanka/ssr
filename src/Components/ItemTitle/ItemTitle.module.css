.ItemTitle {
    padding-top: 2px;
}