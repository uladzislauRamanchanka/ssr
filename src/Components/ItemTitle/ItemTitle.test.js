import React from "react";
import ItemTitle from "./ItemTitle";
import { shallow } from 'enzyme'

describe("Title Component should render properly", () => {
 
  it('should render Title component with props', () => {
    const component = shallow(<ItemTitle name={'Test'}/>)
    expect(component).toMatchSnapshot()
})

it('should render Title component without props', () => {
    const component = shallow(<ItemTitle/>)
    expect(component).toMatchSnapshot()
})
});
