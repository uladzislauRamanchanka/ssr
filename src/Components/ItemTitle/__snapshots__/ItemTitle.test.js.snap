// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render properly should render Title component with props 1`] = `
<div
  className="ItemTitle"
>
  Test
</div>
`;

exports[`Title Component should render properly should render Title component without props 1`] = `
<div
  className="ItemTitle"
>
  Default Title
</div>
`;
